ami/main/api/views.py

```
@@ -1851,6 +1852,59 @@ def get(self, request):
         return Response(data)
 
 
+class UserIdentificationCountsView(GenericAPIView, ProjectMixin):
+    """
+    API endpoint that returns identification counts for top 5 users.
+    Optionally restricted to a specific project via project_id query parameter.
+    """
+
+    permission_classes = [IsActiveStaffOrReadOnly]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)